src/datagen.py

```diff
@@ -231,15 +232,15 @@ def get_weather_feature_lookups(self) -> FeatureLookup:
     def init_training_set(
         self,
         train_data: DataFrame,
-        exclude_columns: List[str] = ["building_id", "upgrade_id", "weather_file_city"],
+        exclude_columns: List[str] = ["building_id", "upgrade_id"],
```

Comment:
Need the weather file city here to merge to the `weather_features_df` and extract the `weather_file_city_index` for prediction in the `src/model_training.py` notebook.